backend/README.md

```diff
@@ -32,11 +32,12 @@ Names and regions should match above.
 ### Deploy
 
 1.  Update `GCS_BUCKET` in `backend/app.yaml`
-2.  Create the files to deploy: `VITE_ON_GCP="true" VITE_RESOURCE_BASE="https://atip-test-2.ew.r.appspot.com/data" npm run build && cd backend && rm -rf dist && cp -R ../dist .`
+2.  Run `gcloud projects describe atip-test-2 | grep projectNumber` and use the result to update `PROJECT_NUMBER` in `backend/app.yaml`
```

Comment:
Annoyingly https://cloud.google.com/appengine/docs/standard/nodejs/runtime#environment_variables gives project ID, but not number